setup.py

```diff
@@ -17,42 +15,9 @@
     pass
```

Comment:
The `cmdclass` variable and this entire block can also go away if you move the docs generation into `tox.ini` ([see example](https://github.com/behave/behave-django/blob/master/tox.ini#L45-L50)). :rocket: